prototypes/matrix_based_advection_diffusion/matrix_based_advection_diffusion.cc

```diff
@@ -1093,9 +1093,13 @@ MatrixBasedAdvectionDiffusion<dim, fe_degree>::assemble_gmg()
 
   for (unsigned int level = 0; level < triangulation.n_global_levels(); ++level)
     {
-      const IndexSet dof_set =
+      const IndexSet active_dofs =
+        DoFTools::extract_locally_active_level_dofs(this->dof_handler, level);
```

Comment:
Do we need the locally active or the locally owned? I thought it would be more like: 
` const IndexSet owned_dofs = this->dof_handler.locally_owned_mg_dofs(level);`

Reply:
I think it's locally owned, I'll switch the function :)!